<ui version="4.0" >
 <class>AppearanceConfigWidget</class>
 <widget class="QWidget" name="AppearanceConfigWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Color &amp; Font Configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Tooltips</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout" >
       <item>
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Show the following information in certificate list tooltips:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="tooltipValidityCheckBox" >
         <property name="text" >
          <string>Show validity</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="tooltipOwnerCheckBox" >
         <property name="text" >
          <string>Show owner information</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="tooltipDetailsCheckBox" >
         <property name="text" >
          <string>Show technical details</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>Certificate Categories</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QListWidget" name="categoriesLV" />
       </item>
       <item row="0" column="1" >
        <layout class="QVBoxLayout" >
         <item>
          <widget class="QPushButton" name="iconButton" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="text" >
            <string>Set &amp;Icon...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="foregroundButton" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="text" >
            <string>Set &amp;Text Color...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="backgroundButton" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="text" >
            <string>Set &amp;Background Color...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="fontButton" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="text" >
            <string>Set F&amp;ont...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="italicCB" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="text" >
            <string>Italic</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="boldCB" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="text" >
            <string>Bold</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="strikeoutCB" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="text" >
            <string>Strikeout</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>20</width>
             <height>100</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="defaultLookPB" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="text" >
            <string>Default Appearance</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
